package.json

```diff
@@ -500,6 +500,16 @@
               "onHover",
               "onSelection"
             ]
+          },
+          "kaoto.colorScheme": {
```

Comment:
picky not:

- `Color Theme` would be more precise if we would like to follow VS Code naming pattern

Reply:
I used the same config setting that we're using in Kaoto UI. We could change the name in VS Code and make the adaptation where the VS Code MA API is created. We might be deviating from the other settings where we use the exact same name, but maybe is worth the small disparity to offer a more VS Code-like experience :+1: 

I'll change it

Reply:
done